tools/src/main/java/org/apache/rocketmq/tools/admin/DefaultMQAdminExtImpl.java

```diff
@@ -222,8 +222,9 @@ public SubscriptionGroupConfig examineSubscriptionGroupConfig(String addr, Strin
     }
 
     @Override
-    public TopicConfig examineTopicConfig(String addr, String topic) {
-        return null;
+    public TopicConfig examineTopicConfig(String addr, String topic) throws RemotingException, InterruptedException, MQBrokerException {
+        TopicConfigSerializeWrapper topicConfigSerializeWrapper = this.mqClientInstance.getMQClientAPIImpl().getAllTopicConfig(addr,timeoutMillis);
+        return topicConfigSerializeWrapper.getTopicConfigTable().get(topic);
```

Comment:
check topicConfigSerializeWrapper.getTopicConfigTable().get(topic) is null and throw Exception?